databricks_cli/stack/api.py

```diff
@@ -139,6 +162,84 @@ def deploy_config(self, stack_config, stack_status=None, **kwargs):
 
         return new_stack_status
 
+    def download_config(self, stack_config, **kwargs):
+        """
+        Downloads a stack given a dict of the stack configuration
+        :param stack_config:
+        :param kwargs:
+        :return:
+        """
+        self._validate_config(stack_config)
+        click.echo('#' * 80)
+        for resource_config in stack_config.get(STACK_RESOURCES):
+            # Deploy resource, get resource_status
+            self._download_resource(resource_config, **kwargs)
+            click.echo('#' * 80)
+
+    def _download_resource(self, resource_config, **kwargs):
+        """
+        Downloads a resource given a resource information extracted from the stack JSON
+        configuration template.
+
+        :param resource_config: A dict of the resource with fields of RESOURCE_ID, RESOURCE_SERVICE
```

Comment:
In the documents you should use "resource_id" instead of the variable `RESOURCE_ID`. Does python do the replacements automatically?

Reply:
Wish it did, but I put `RESOURCE_ID` since it's inline with the code. I'll switch to 'resource_id' and so on to reduce confusions. I realize this documentation should be more user-readable than developer-readable.